app/DynamicsAdapter/DynamicsAdapter.Web/appsettings.json

```diff
@@ -7,6 +7,32 @@
     }
   },
   "AllowedHosts": "*",
+  "AppSettings": {
+    "SchedulerConfig": {
+      "Cron": "0/5 * * * * ?"
+    },
+    "DynamicsAPIConfig": {
+      "Timeout": 0,
+      "OAuthUrl": "",
+      "ResourceUrl": "",
+      "ClientId": "",
+      "Secret": "",
+      "Username": "",
+      "Password": "",
+      "TokenTimeout": 0,
+      "EndPoints": [
+        {
+          "Entity": "",
+          "EndPoint": ""
+        },
+        {
+          "Entity": "",
+          "EndPoint" :  "" 
+        }
+      ] 
+    }
+  }
+   
   "Scheduler": {
```

Comment:
needs to be removed ;-)

Reply:
I wanted to move them app under appsettings section? I am not sure what is required here..

Reply:
oh just to have it under appSettings